grakn-graql/src/main/java/ai/grakn/graql/internal/util/CommonUtil.java

```diff
@@ -48,6 +49,18 @@
 
     private CommonUtil() {}
 
+    public static <T> T withImplicitConceptsVisible(GraknGraph graph, Supplier<T> function) {
```

Comment:
Might be worth using these methods across the whole stack. I.e. put it in grakn-core and use it everywhere.